src/coreclr/binder/assemblyname.cpp

```diff
@@ -12,18 +12,19 @@
 // ============================================================
 
 #include "assemblyname.hpp"
-#include "assembly.hpp"
 #include "utils.hpp"
-#include "variables.hpp"
-
-#include "fusionassemblyname.hpp"
 
 #include "textualidentityparser.hpp"
 
 #include "corpriv.h"
 
 #include "ex.h"
 
+namespace
+{
+    const WCHAR* s_neutralCulture = W("neutral");
```

Comment:
Nit: Is this constant documented somewhere?

Reply:
This is the only thing I found: https://docs.microsoft.com/dotnet/framework/reflection-and-codedom/specifying-fully-qualified-type-names#specifying-assembly-names

Reply:
Perhaps reference that? Just want to make sure that is a known reserved word.